inlang/source-code/paraglide/paraglide-js/src/compiler/compileMessage.ts

```diff
@@ -156,29 +143,63 @@ const messageFunction = (args: {
  */
 /* @__NO_SIDE_EFFECTS__ */
 export const ${args.message.id} = (${hasParams ? "params" : ""}) => ${args.compiledPattern}
-${
-	args.message.alias["default"] &&
-	args.message.id !== args.message.alias["default"] &&
-	` 
-/**
- * This message is an alias of ${args.message.id}
- */
-export const ${args.message.alias["default"]} = ${args.message.id};
-`
-}
+${reexportAliases(args.message)}
 `
 }
 
-function reexportMessage(messageId: string, fromLanguageTag: string) {
-	return `export { ${messageId} } from "./${fromLanguageTag}.js"`
+function reexportMessage(message: Message, fromLanguageTag: string) {
+	const exports: string[] = [message.id]
+
+	if (message.alias["default"] && message.id !== message.alias["default"]) {
+		exports.push(message.alias["default"])
+	}
+
+	return `export { ${exports.join(", ")} } from "./${fromLanguageTag}.js"`
 }
 
-function messageIdFallback(messageId: string, languageTag: string) {
+function messageIdFallback(message: Message, languageTag: string) {
 	return `/**
-* Failed to resolve message ${messageId} for languageTag "${languageTag}". 
+* Failed to resolve message ${message.id} for languageTag "${languageTag}". 
 * @returns {string}
 */
 /* @__NO_SIDE_EFFECTS__ */
-export const ${messageId} = () => "${escapeForDoubleQuoteString(messageId)}"
+export const ${message.id} = () => "${escapeForDoubleQuoteString(message.id)}"
+${reexportAliases(message)}
+`
+}
+
+/**
+ * Returns re-export statements for each alias of a message.
+ * If no aliases are present, this function returns an empty string.
+ *
+ * @param message
+ */
+function reexportAliases(message: Message) {
+	let code = ""
+
+	if (message.alias["default"] && message.id !== message.alias["default"]) {
+		code += `
+/**
+ * Change the reference from the alias \`m.${message.alias["default"]}()\` to \`m.${message.id}()\`:
+ * \`\`\`diff
+ * - m.${message.alias["default"]}()
+ * + m.${message.id}()
+ * \`\`\`
+ * ---
+ * \`${message.alias["default"]}\` is an alias for the message \`${message.id}\`.
+ * Referencing aliases instead of the message ID has severe downsides like:
```

Comment:
I would remove the word `severe`

> Referencing aliases instead of the message ID has ~~severe~~ downsides like:

Reply:
agree. can you also change "edit in the web editor" to "edit in fink"?